src/components/plugin/glossary-sidebar.tsx

```diff
@@ -104,15 +103,10 @@ export default class GlossarySidebar extends React.Component<IProps, IState> {
     }
     return (
       <div className={css.glossarySidebar}>
-        {
-          secondLanguage && onLanguageChange &&
-          <Button
-            data-cy="langToggle"
-            className={css.langButton}
-            label={POEDITOR_LANG_NAME[secondLanguage].replace("_", " ")}
-            onClick={onLanguageChange}
-          />
-        }
+        <LanguageSelector
+          otherLanguages={otherLanguages}
+          onLanguageChange={onLanguageChange}
+        />
```

Comment:
Same potential issue here as above, that the language select looks like it is shown all of the time, but I haven't made it to that component yet...

Reply:
I see it is handled in the component itself.